tests/manager/manager-ctor-001.phpt

```diff
@@ -1,12 +1,12 @@
 --TEST--
-MongoDB\Driver\Manager::__construct()
+MongoDB\Driver\Manager::__construct() with default URI
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
 --FILE--
 <?php
 require_once __DIR__ . "/../utils/basic.inc";
 
-$manager = new MongoDB\Driver\Manager(STANDALONE);
+$manager = new MongoDB\Driver\Manager();
```

Comment:
I don't think this is going to work. The STANDALONE still points to a PHONGO SERVERS.json link - and "localhost" isn't always available is it? In any case, you shouldn't modify an existing case that doesn't change behaviour with the patch.